src/clj/tasks/nrdb.clj

```diff
@@ -61,13 +61,17 @@
   (let [c (filter #(contains? % :previous-versions) cards)]
     (reduce expand-card `() c)))
 
+;; these are cards with multiple faces, so we can't download them directly
+(def ^:const cards-to-skip #{"08012" "09001" "26066" "26120"})
+
 (defn download-card-images
   "Download card images (if necessary) from NRDB"
   [cards]
   (let [img-dir (io/file "resources" "public" "img" "cards" "en" "default" "stock")]
     (io/make-parents img-dir)
     (let [previous-cards (generate-previous-card-stubs cards)
           total-cards (concat cards previous-cards)
+          total-cards (remove #(get cards-to-skip (:code %)) total-cards)
```

Comment:
This won't download those card images at all. Is this intended?

Reply:
We don't want to download anything for the cards with multiple faces. Since we don't have a way to get all the faces. Currently those are in the repo. The way the image code works, it will prioritize a non-face image over a face image, so we don't want those in the path.

Reply:
Okay. You'll have to explain it in greater detail at some point.